syntax = "proto3";

message Person {
  string name = 1;
  string account_no = 2;
  string pin_no = 3;
  int32 amount = 4;
}

message BankAccounts {
  repeated Person person = 1;
}
